### PR TITLE
Poll current connections for F5 ltm

### PR DESCRIPTION
**Pull request by martinberg**
Adds polling for "current connections" for LTM VS:es and LTM Pools.
Unfortunately this adds a new DS to a already existing RRD-file and therefore the RRD-file needs to be recreated.

bild

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by PipoCanaja** [comment]
Hi @martinberg 
Any chance you could create a new RRD file instead of extending an existing one ?

**Comment by martinberg** [comment]
> 
> 
> Hi @martinberg
> Any chance you could create a new RRD file instead of extending an existing one ?

I'm unsure how to accomplish that in a way that would make sense in case somebody wants to add more stats in the future. 
I feel that `includes/polling/loadbalancers/f5-ltm.inc.php` is the right place to loop through all objects at least but don't know how to implement it.
Do you have any suggestions or are able to put together something?

**Comment by murrant** [comment]
Definitely make a new RRD.  If I could redo anything in LibreNMS, I would only put 1 DS per rrd... that way you can and and remove values without breaking everything.

**Comment by martinberg** [comment]
Made a separate RRD. Not sure if this is a good way to do it but this is what I came up with.
Let me know if there is a better way :)

File: includes/html/graphs/device/bigip_ltm_allpm_currconns.inc.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?php
+/*
+ * LibreNMS module to display F5 LTM Virtual Server Details
+ *
+ * Copyright (c) 2016 Aaron Daniels <aaron@daniels.id.au>
```
Comment (PipoCanaja) [inline includes/html/graphs/device/bigip_ltm_allpm_currconns.inc.php:5]: Hi
Could you please correct the copyright on files ?
Thanx

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

File: includes/html/graphs/device/bigip_ltm_allpm_currconns.inc.php (hunk 1 above)
Comment (martinberg) [reply includes/html/graphs/device/bigip_ltm_allpm_currconns.inc.php]: Thanks! Forgot about that part

**Review by PipoCanaja — APPROVED** [review]
LGTM

**Pull request merged**